examples/server/server.cpp

```diff
@@ -852,7 +852,19 @@ struct server_context {
         // infill
         slot.params.input_prefix = json_value(data, "input_prefix", default_params.input_prefix);
         slot.params.input_suffix = json_value(data, "input_suffix", default_params.input_suffix);
-        slot.prompt              = json_value(data, "prompt",       std::string(""));
+
+        // get prompt
+        {
+            const auto & prompt = data.find("prompt");
+            if (prompt == data.end())
+            {
+                slot.prompt = "";
+            }
+            else
+            {
+                slot.prompt = *prompt;
+            }
```

Comment:
We recently updated the coding style in `server` to match the rest of the codebase, so now opening braces are on the same line:

```suggestion
            const auto & prompt = data.find("prompt");
            if (prompt == data.end()) {
                slot.prompt = "";
            } else {
                slot.prompt = *prompt;
            }
```